recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: cobra
+  version: "0.4.1"
+
+source:
+  fn: cobra-0.4.1.tar.gz
+  url: https://pypi.python.org/packages/source/c/cobra/cobra-0.4.1.tar.gz
+  md5: a96799117abed427333bb57e007a8b82
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - glpk
+
+  run:
+    - python
+    - six
+    - numpy
+    - scipy
+    - lxml
+    #- python-libsbml
+    - glpk
+
+test:
+  imports:
+    - cobra
+    - cobra.core
+    - cobra.design
+    - cobra.flux_analysis
+    - cobra.io
+    - cobra.manipulation
+    - cobra.solvers
+    - cobra.test
+    - cobra.topology
+
+  requires:
+    - jsonschema
+
+about:
+  home: https://opencobra.github.io/cobrapy
+  license: GNU Lesser General Public License v2 or later (LGPLv2+) or GNU General Public License v2 or later (GPLv2+)
+  summary: 'COBRApy is a package for constraint-based modeling of biological networks'
+
+extra:
+  recipe-maintainers:
+    - aebrahim
+    - vivekiitkgp
```

Comment:
I am sure you guys talked, but just to confirm. Are you ok with being a maintainer on this recipe, @vivekiitkgp?